statuspage/src/utils/Config.default.js

```diff
@@ -19,7 +19,8 @@ export const Config = {
     'en': 'English',
     'de': 'Deutsch',
     'it': 'Italiano',
-    'zh_TW': '���餤��'
+    'zh_TW': '���餤��',
```

Comment:
Could you check why this got corrupted? (Double-check you're using UTF-8 when saving?)

Reply:
Actually this was already corrupt before. I fixed it in `master`. Could you rebase your change?